src/OpenLoco/src/Paint/Paint.cpp

```diff
@@ -1007,6 +1008,17 @@ namespace OpenLoco::Paint
         }
     }
 
+    static bool isTypeCullableBridge(const Ui::ViewportInteraction::InteractionItem type)
+    {
+        switch (type)
```

Comment:
since we don't check any more interaction items, and it seems unlikely that many more will be added to this list soon (correct me if i'm wrong though), then we don't really need the switch here, you can just do `return type == Ui::ViewportInteraction::InteractionItem::bridge;`

Reply:
Done